apps/omg_watcher_info/lib/omg_watcher_info/api/transaction.ex

```diff
@@ -126,6 +188,78 @@ defmodule OMG.WatcherInfo.API.Transaction do
 
     outputs
     |> Stream.concat(Stream.repeatedly(empty_gen))
-    |> (&Enum.zip([:output0, :output1, :output2, :output3], &1)).()
+    |> (fn output -> Enum.zip([:output0, :output1, :output2, :output3], output) end).()
+  end
+
+  defp create_transaction(utxos_per_token, %{
+         owner: owner,
+         payments: payments,
+         metadata: metadata,
+         fee: fee
+       }) do
+    rests =
```

Comment:
please don't break down arguments in function headers in there are no multiple function headers that you need to pattern match.